### PR TITLE
Nymph changes/fixes

### PR DESCRIPTION
**Pull request by Yonnimer**
- The bug causing the nymph health hud to always been low has been fixed.

- Nymphs now have a new health hud, and the "fire" hud was removed.

### CODE REVIEW

**Comment by Caelphon** [comment]
Nymphs quite literally are capable of surviving planetary re-entry, I don't see why their health would be only a few notches above a rat. 

+1

**Comment by Yonnimer** [comment]
!review

**Comment by ParadoxSpace** [comment]
No to the health increase. A pile of nymphs on the floor do not need more health than them collected into a gestalt.

**Comment by Yonnimer** [comment]
Nymphs are also borderline useless when it comes to doing anything. Increasing their health up a bit is not an issue.

**Comment by Yonnimer** [comment]
!wip

**Comment by Yonnimer** [comment]
Removing to WIP while another pr is being made to meet arrow's requirement.

**Comment by Chada-1** [comment]
@Yonnimer One way you might be able to meet a middle ground with arrow/paradox/geeves is to try to make a way for singular grown Nymphs to be somehow different than the kind of nymphs you get when you split, so they can actually have more health than the other kinds.

**Comment by TehFlaminTaco** [comment]
Suggest giving nymphs internal armour rather than increasing their health  to better match the re-entry survival properties.

**Comment by TheDilShawn** [comment]
Perhaps I can suggest making the health of nymphs without much biomass (so new nymphs) really weak but as their biomass goes up it becomes more resillient and with more HP, more intune with the lore?

**Comment by Yonnimer** [comment]
Nymph health will be worked on in another pr, so that has been removed from this one. I've also changed how the hud detects nymph health, so it shouldn't need to be changed anymore if nymph health changes.

**Comment by Yonnimer** [comment]
!review

File: code/_onclick/hud/alien_larva.dm [hunk 1]
```diff
@@ -16,7 +16,7 @@
 	move_intent = using
```
Comment (NonQueueingMatt) [inline code/_onclick/hud/alien_larva.dm:16]: This hud is still named larva_hud, you should change it to diona_hud at this rate

File: code/modules/mob/living/carbon/alien/life.dm [hunk 2]
```diff
@@ -108,7 +108,7 @@
 
 	if (healths)
 		if (stat != 2)
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/carbon/alien/life.dm:110]:
```suggestion
		if (stat != DEAD)
```

**Comment by Yonnimer** [comment]
hud is no longer showing up for nymphs, can't figure out why. Might just be my byond being a butt though.

**Comment by Yonnimer** [comment]
It's working, just my byond was being a butt.

**Pull request merged**